### PR TITLE
GEODE-7156: add docs for security-auth-token-enabled-components

### PR DESCRIPTION
**Pull request by jinmeiliao**
Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, check Concourse for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

File: geode-docs/configuring/running/cluster-management-service.html.md.erb [hunk 1]
```diff
@@ -73,3 +73,19 @@ This produces a browsable description of the cluster management service API:
...
+
+The cluster management service REST API is secured by the <code class="ph codeph">security-manager</code> of your cluster. If a security-manager is specified, by default, you will
+need to pass in a username/password pair in order to invoke the rest api.
+
+```
+curl --user username:password http://example.com/management/regions
+```
+
+When <code class="ph codeph">security-auth-token-enabled-components</code> is set to "all" or "management", you will
+need to pass in a valid bear token in your request header in order to invoke the rest api.
```
Comment (dschneider-pivotal) [inline geode-docs/configuring/running/cluster-management-service.html.md.erb:87]: change "bear" to "bearer"

File: geode-docs/reference/topics/gemfire_properties.html.md.erb [hunk 2]
```diff
@@ -512,12 +512,23 @@ Any security-related (properties that begin with <code class="ph codeph">securit
 <td><em>not set</em></td>
 </tr>
 <tr>
-<td>security-client-accessor</td>
+<td>security-auth-token</td>
```
Comment (dschneider-pivotal) [inline geode-docs/reference/topics/gemfire_properties.html.md.erb:515]: Why did you change "security-client-accessor" to "security-auth-token"? I don't see the latter in the product but the client-accessor one exists

File: geode-docs/reference/topics/gemfire_properties.html.md.erb [hunk 3]
```diff
@@ -512,12 +512,23 @@ Any security-related (properties that begin with <code class="ph codeph">securit
...
 <td><b>Deprecated.</b> Used for authorization. Static creation method returning an <code class="ph codeph">AccessControl</code> object, which determines authorization of client-server cache operations. This specifies the callback that should be invoked in the pre-operation phase, which is when the request for the operation is received from the client.</td>
 <td>S, L</td>
 <td><em>not set</em></td>
 </tr>
 <tr>
+<td>security-auth-token-enabled-components</td>
+<td>A comma delimited list of component names which works in conjunction with the <code class="ph codeph">security-manager</code> property. if security manager is enabled, this property will determine what rest end point will use token based authentication instead of basic (username/password) authentication.
```
Comment (dschneider-pivotal) [inline geode-docs/reference/topics/gemfire_properties.html.md.erb:522]: instead of "what rest end point" I think "what components" is better.

File: geode-docs/reference/topics/gemfire_properties.html.md.erb [hunk 4]
```diff
@@ -518,6 +518,17 @@ Any security-related (properties that begin with <code class="ph codeph">securit
 <td><em>not set</em></td>
 </tr>
 <tr>
+<td>security-auth-token-enabled-components</td>
+<td>A comma delimited list of component names which works in conjunction with the <code class="ph codeph">security-manager</code> property. if security manager is enabled, this property will determine what components will use token based authentication instead of basic (username/password) authentication.
+<p>Valid values are: "all", "management", "pulse"</p>
```
Comment (dschneider-pivotal) [inline geode-docs/reference/topics/gemfire_properties.html.md.erb:523]: do literal quotes work here or do you need to use &quot;

Comment (jinmeiliao) [reply geode-docs/reference/topics/gemfire_properties.html.md.erb:523]: they show up correctly in the generated docs
